docs/UserGuide.md

```diff
@@ -279,11 +541,30 @@ Format: `timetable -del <day> <lesson index>`
 Example of usage: 
 * `timetable -del MONDAY 1`
 
+Example of output:
+```
+Command: 
+⋗	timetable -del MONDAY 1
+════════════════════════════════════════════════════════════════════════════════
+Your lesson has been deleted successfully.
+
+════════════════════════════════════════════════════════════════════════════════
+```
```

Comment:
Would be wise to show the Lesson that has been removed

Reply:
This one will need to be changed in the program itself too @Aseanseen